src/cpp/src/tokenizer/tokenizer.cpp

```diff
@@ -208,6 +209,11 @@ class Tokenizer::TokenizerImpl {
     // handle to shared_object with openvino tokenizers
     std::shared_ptr<void> m_shared_object_ov_tokenizers = nullptr;
 
+    // buffer to store the error messages from the pass
+    std::ostringstream m_pass_errors;
```

Comment:
`m_pass_errors` is not used anymore, I guess we can remove it
```suggestion
```

Reply:
done